rpc/rest/README.md

```diff
@@ -47,3 +47,83 @@ $ curl -X GET http://localhost:9191/example
   "Example": "This is an example"
 }
 ```
+
+
+**Security**
```

Comment:
These parts of doc should use sub-header text (`##`, `###`) instead of simple bold. That way it can be linked to.